grafana/DOCS.md

```diff
@@ -40,8 +40,8 @@ Example add-on configuration:
 log_level: info
 grafana_ingress_user: frenck
 plugins:
-  - ayoungprogrammer-finance-datasource
-  - grafana-clock-panel
+  - name: ayoungprogrammer-finance-datasource
+  - name: grafana-clock-panel
```

Comment:
I'm not understanding why this has to be a breaking change?

Can't we accept both variants?

If not, I'm leaning towards not supporting custom plugins (as always has been the case).

../Frenck

Reply:
We can accept... and I tried... Just that I didn't know how to proper do the config.yaml. Any help so I can try again?

Reply:
It's not a breaking change anymore. I reverted the plugin section and added a new custom_plugins one with the possibility to have an url and an unsigned flag.